.github/scripts/run_on_all_chains.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+# Disable immediate exit on non-zero status
+set +e
+
+command=$1
+chain_list=$2
+log_dir=$3
+IFS=',' read -r -a chains <<< "$chain_list"
+pids=()
+statuses=()
+
+# Start background processes
+for i in "${!chains[@]}"; do
+    eval "$command --chain ${chains[$i]} &> ${log_dir}/${chains[$i]}.log" &
+    pids+=($!)
+done
+
+# Wait for all processes to complete and capture their exit statuses
+for i in "${!pids[@]}"; do
+    wait ${pids[$i]}
+    statuses[$i]=$?
+done
+
+# Check exit statuses and set overall status
+overall_status=0
+
+for i in "${!statuses[@]}"; do
+    if [ ${statuses[$i]} -ne 0 ]; then
+        overall_status=1
+        echo "Chain ${chains[$i]} failed with status ${statuses[$i]}"
```

Comment:
Could we add colors? And make it green for success and red for failed?

Reply:
Nice! Addressed in https://github.com/matter-labs/zksync-era/pull/3025/commits/68967ea06cf7e9b8a5890d18f4df4adbaa0ad032